src/app/pages/component-sidenav/component-nav.html

```diff
@@ -0,0 +1,20 @@
+<div class="docs-component-viewer-nav">
+  <div class="docs-component-viewer-nav-inner">
+    <nav *ngFor="let category of docItems.getCategories((params | async)?.section); let last = last;">
+      <h3 (click)="toggleExpand(category.id)">
```

Comment:
For a11y, each of these sections needs to act like an expansion panel. This means that the thing you interact with should be a `<button>` (with the same aria attrs that expansion panel uses). We shouldn't need to use a real heading element here since it's not part of the document content (table of contents).